src/Middleware/RequestBodyBuffer.php

```diff
@@ -61,6 +77,6 @@ private function iniMaxPostSize()
             return substr($size, 0, -1) * 1024  * 1024 * 1024;
         }
 
-        return $size;
+        throw new \UnexpectedValueException('post_max_size value is out of range.');
```

Comment:
This will now throw for a size specified in bytes (without a suffix)

Reply:
Good point, I've reverted it